systemtest/src/main/java/io/strimzi/systemtest/utils/specific/ContainerRuntimeUtils.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.utils.specific;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+public class ContainerRuntimeUtils {
+    private static final String DOCKER = "docker";
+    private static final String PODMAN = "podman";
+    private static final String RUNTIME = detectRuntime();
+
+    private static String detectRuntime() {
+        if (isCommandAvailable(PODMAN)) {
+            return PODMAN;
+        } else if (isCommandAvailable(DOCKER)) {
+            return DOCKER;
```

Comment:
I was thinking about using something like this https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/ (`kubectl get nodes -o wide`) where the container runtime is specified. But for Podman it seems to be `containerd` (at least from what I tried).
And I guess that would not be helpful.

Reply:
Yeah, that would not help us :/.